projects/flop_count_example/Flop_Count_Example.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "82047528",
+   "metadata": {},
+   "source": [
+    "# NeuralCompression Flop Counter Example\n",
+    "\n",
+    "Welcome! In this notebook we'll walkthrough using `neuralcompression`'s flop counter to calculate the computational complexity of a compression model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "24a69e05",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING:absl:No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import torch\n",
+    "from torch import nn\n",
+    "\n",
+    "from neuralcompression.functional import count_flops\n",
```

Comment:
Please use `import neuralcompression.functional as ncF` in all docs - I'd like to keep this the standard style.